README.md

```diff
@@ -19,7 +19,7 @@ export MSG_SIZE_KBYTES=10
 export PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 export ETH_FROM=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
 export RLN_RELAY_EPOCH_SEC=10
-export RLN_RELAY_MSG_LIMIT=2
+export RLN_RELAY_MSG_LIMIT=2  # Currently the value must be <= 20, as that's the contract currently hardcoded maximum.
```

Comment:
note that 20 is hardcoded in https://github.com/waku-org/waku-rlnv2-contract/blob/main/script/Deploy.s.sol#L16 and will soon be configurable. cc @stubbsta 

So I suggest having it as configurable, and then documenting here how to set your own. eg
export RLN_MAX_MSG_LIMIT=x